### DISCUSSION TITLE
A video on IPv4 IP addresses

### DISCUSSION THREAD

**Discussion by 737simpilot** [CIDRAM/CIDRAM#301] [General]
I follow this guy's YouTube channel and I see he recently uploaded this nice video on IPv4 IP addresses. So, I thought it might be of interest to some here. Wait till the end about loopback. You'll love it. Or maybe not... :flushed:

https://youtu.be/tcae4TSSMo8

**Comment by Maikuolan** [comment]
Although a subject that you and I both are already very familiar with, one which perhaps many others are not, and the guy provides the relevant information in a well-presented, clear, easy-to-understand manner, and with just enough humour thrown in to keep it interesting without overdoing it. A good video. Thanks for sharing. :-)
